siliconcompiler/core.py

```diff
@@ -310,7 +310,7 @@ def create_cmdline(self, progname, description=None, switchlist=None, input_map=
                                          description=description)
 
         # Get all keys from global dictionary or override at command line
-        allkeys = self.getkeys()
+        allkeys = self.allkeys()
```

Comment:
Is the intent to make allkeys() a public method now and if so, does that completely replace the getkeys() behavior (or do we now have two ways of getting all the keys back?)

Reply:
Nope, two ways that do different things (see new comment summarizing methods)